prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -98,6 +98,14 @@ class FhirToHl7Converter(
         // Add any schema level constants to the context
         // We need to create a new context, so constants exist only within their specific schema tree
         val schemaContext = CustomContext.addConstants(schema.constants, context)
+
+        // check for duplicate named schema elements before processing
+        val dupes =
```

Comment:
You want to do this check while validating the schema and not when processing data.

Reply:
Moved the check up the chain

Reply:
I do not see the move.  It is still done in the convert.  SHould be done as part of the schema validate.